src/pages/derivx/_hero.js

```diff
@@ -0,0 +1,36 @@
+import React, { useState, useEffect } from "react"
+import { Localize } from 'components/localization'
+import DHero from 'components/custom/_dhero-2'
+import { size } from 'themes/device'
+import DerivXBG from 'images/svg/derivx/triangle-up.svg'
+import DerivXBG2 from 'images/svg/derivx/triangle-down.svg'
+import derivx_logo from 'images/svg/derivx/derivx-logo.svg'
+import { isBrowser } from 'common/utility'
+
+const Hero = () => {
+    const [is_mobile, setMobile] = useState(false)
+    const handleResizeWindow = () => {
+        setMobile(isBrowser() ? window.screen.width <= size.mobileL : false)
+    }
+
+    useEffect(() => {
+        setMobile(isBrowser() ? window.screen.width <= size.mobileL : false)
```

Comment:
```suggestion
        handleResizeWindow()
```
And also the useEffect() deps + useCallback wrap to the event listener function. Also cleanup function~